sphire/protocols/protocol_cryolo_picking.py

```diff
@@ -251,7 +251,7 @@ def getInputModel(self):
         return os.path.abspath(m) if m else ''
 
     def getOutputDir(self):
-        return self._getTmpPath('outputEMAN')
+        return self._getExtraPath('Boxfiles')
```

Comment:
Hi @the-best-elephant , why do you want to keep in 'extra' the temporary box files? They will be parsed and the SetOfCoordinates generated, so I think it is not needed.

Reply:
Hi @delarosatrevin , we were considering the box file not temporary, but the actual output of Cryolo (same as relion star files output, or xmipp). As such, we are doing the same, keeping the output of the program. Shouldn't we?

Reply:
It is a temporary output that is parsed and no longer needed. I'm going the other way around in Relion and other plugins...removing files not needed.